docs/en/enterprise-edition/content-collections/runtime-security/agentless-scanning/configure-accounts/frag-start-agentless-scan.adoc

```diff
@@ -1,27 +1,65 @@
-[#start-agentless-scan]
+[#agentless-scanning-for-cloud-accounts]
 [.task]
-== Start an Agentless Scan
+=== Agentless Scanning for Cloud Accounts
 
-Agentless scans start immediately after onboarding the cloud account.
-By default, agentless scans are performed every 24 hours, but you can change the interval on the *Manage > System > Scan* page under *Scheduling > Agentless*.
+When new cloud accounts are onboarded to Prisma Cloud with the "Agentless Scanning" option enabled, scanning starts immediately. If this option is disabled before onboarding, scanning does not occur for those accounts.
+
+When you enable agentless scanning for existing accounts in Prisma Cloud, it does not trigger an immediate scan. Instead, the account is included in the next scan cycle, which occurs every 24 hours by default.
+
+To modify the scan cycle, see <<modifying-the-agentless-scan-interval>>
+
+[#editing-agentless-scan-settings]
+=== Editing Agentless Scan Settings
+
+For existing cloud accounts that have agentless scan settings disabled, you can now modify the settings directly in Runtime Security > Manage > Cloud accounts page. 
+
+To edit agentless scan settings, complete the following steps:
+
+[.procedure]
+. Go to *Runtime Security > Manage > Cloud accounts*.
+
+. Click the *Edit Account* icon from the Actions column for the account. 
+
+. In the *Account and Agentless setup* dialog box, go to *Agentless scanning* section.
+
+. Modify the agentless configuration options in this section. 
+
+ . Click *Save*.
+
+After the configuration is modified, the next scan uses the updated settings.
+
+[#modifying-the-agentless-scan-interval]
```

Comment:
consider keeping the same hook to avoid breaking places that link to this section, or edit other places for this updated hook

Reply:
I have retained the old hook [#start-agentless-scan] for the "manually starting an agentless can" section.